dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/utils/OpenAPITestUtil.java

```diff
@@ -237,8 +241,8 @@ public static void checkInfo(JsonNode root, String defaultTitle, String defaultV
         String title = infoNode.get("title").textValue();
```

Comment:
Aren't `title` and `version` constants somewhere?

Reply:
Oddly, it doesn't look like they are. Usually we have a model object and we're calling `Info.getTitle()` instead, or we're not in java code and can't use a constant anyway.